### PR TITLE
Change 1.0 ref to 1.X

### PR DESCRIPTION
**Pull request by AreYouRlyScared**
Changes 1.0 references to 1.X as it's causing confusion.

### CODE REVIEW

File: daemon/0.6/installing.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Installation
 
 ::: warning
-This specific software is for Pterodactyl v0.7 and **must not be used for Pterodactyl v1.0**. If you have installed 1.0 you should use [Wings](/wings/1.0/installing.html) instead.
+This specific software is for Pterodactyl v0.7 and **must not be used for Pterodactyl v1.X**. If you have installed 1.0 you should use [Wings](/wings/1.0/installing.html) instead.
```
Comment (TekExplorer) [inline daemon/0.6/installing.md:4]: missed one
```suggestion
This specific software is for Pterodactyl v0.7 and **must not be used for Pterodactyl v1.X**. If you have installed 1.X you should use [Wings](/wings/1.0/installing.html) instead.
```

**Comment by DaneEveritt** [comment]
This breaks production links to content because you renamed files.

**Comment by AreYouRlyScared** [comment]
image

**Comment by AreYouRlyScared** [comment]
No longer has broken links 👍

**Review by DaneEveritt — APPROVED** [review]

**Comment by DaneEveritt** [comment]
This should not have package-lock changes. Please delete `package-lock.json` (no clue why it ended up here, whoops) and ensure you're using `yarn` for all commands.

**Comment by AreYouRlyScared** [comment]
Hmm that happened when I updated the browser list for caniuse

**Comment by TekExplorer** [comment]
just gonna mention #308 due to relavence

**Pull request closed without merging**